client-app/.babelrc

```diff
@@ -1,6 +1,3 @@
 {
-    "presets": [
-        "@babel/preset-env",
-        "@babel/preset-react"
-    ]
-}
```

Comment:
@Gayatrinj Please revert the changes to this file. I would suggest to not push any trivial changes.